### PR TITLE
feat(detection): add ability to check for volume control support

### PR DESCRIPTION
**Pull request by virtuacoplenny**
I did some mind-changing with naming mid-way through but maybe it works out better this way.

### CODE REVIEW

File: spot-client/src/common/app-state/spot-tv/selectors.js [hunk 1]
```diff
@@ -57,3 +57,14 @@ export function getRemoteSpotTVRoomName(state) {
...
 }
+
+/**
+ * A selector which returns whether or not the Spot-TV has volume control
+ * support.
+ *
+ * @param {Object} state - The Redux state.
+ * @returns {boolean}
+ */
+export function isVolumeControlSupported(state) {
```
Comment (paweldomas) [inline spot-client/src/common/app-state/spot-tv/selectors.js]: is that used anywhere ?

Comment (virtuacoplenny) [reply spot-client/src/common/app-state/spot-tv/selectors.js]: Not right now. It's for the volume control work that should be landing this week.

Comment (paweldomas) [reply spot-client/src/common/app-state/spot-tv/selectors.js]: Can we land it with that work as first commit in the PR ?

Comment (virtuacoplenny) [reply spot-client/src/common/app-state/spot-tv/selectors.js]: That’s fair. I’ll leave it I zoli.

**Pull request closed without merging**